tools/benchmark_tool/setup.py

```diff
@@ -6,92 +6,18 @@
 """
 Use this script to create a wheel with OpenVINO™ Python* tools:
 
-$ python setup.py sdist bdist_wheel
+$ python -m build
```

Comment:
folder?

Reply:
Changed to `python -m pip wheel .` as suggested in other comment.